src/NRedisStack/Auxiliary.cs

```diff
@@ -132,23 +131,7 @@ public async static Task<List<RedisResult>> ExecuteBroadcastAsync(this IDatabase
 
         public static string GetNRedisStackVersion()
         {
-            XDocument csprojDocument = GetCsprojDocument();
-
-            // Find the Version element and get its value.
-            var versionElement = csprojDocument.Root!
-                .Descendants("Version")
-                .FirstOrDefault();
-
-            return versionElement!.Value;
-        }
-
-        private static XDocument GetCsprojDocument()
-        {
-            string csprojFilePath = Path.Combine(Directory.GetCurrentDirectory(), "..", "..", "..", "..", "..", "src", "NRedisStack", "NRedisStack.csproj");
-
-            // Load the .csproj file.
-            var csprojDocument = XDocument.Load(csprojFilePath);
-            return csprojDocument;
+            return typeof(NRedisStack.Auxiliary).Assembly.GetName().Version.ToString();
```

Comment:
@slorello89  Thanks for noticing, I actually already fixed it and probably deleted it by mistake.
This line specifically will return a version of type: x.y.z.x no?
Because I want it to be x.y.z, without the fourth digit.